src/ClassicUO.Assets/MapLoader.cs

```diff
@@ -415,7 +407,7 @@ public unsafe bool ApplyPatches(ref StackDataReader reader)
         {
             ResetPatchesInBlockTable();
 
-            PatchesCount = (int) reader.ReadUInt32BE();
+            int PatchesCount = (int) reader.ReadUInt32BE();
```

Comment:
Nit: Local variables should be camel cased.

Reply:
OK, I'll change it. I wanted to keep the diff small, so I avoided using a different name.